goc/share/generate_sshauthkeys

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+
+import sys
+import os
+import pwd, grp
+import re
+import subprocess
+import json
+
+# it's much easier to switch to curl for the authenticated lookup than
+#  to use native python
+cmd = 'curl -sS --cert /etc/grid-security/hostcert.pem --key /etc/grid-security/hostkey.pem "https://topology.opensciencegrid.org/oasis-managers/json?vo=*"'
+p = subprocess.Popen( ('/bin/bash', '-cx', cmd), stdout=subprocess.PIPE)
```

Comment:
Note that the curl command-line will be printed to stderr, presumably that's what you want

Reply:
Yes it is.